features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -345,14 +345,14 @@ Feature: SDN compoment upgrade testing
     When I use the "policy-upgrade5" project
     Given I obtain test data file "networking/list_for_pods.json"
     When I run oc create over "list_for_pods.json" replacing paths:
-      | ["items"][0]["spec"]["replicas"]                      | 1                       |
-      | ["items"][0]["spec"]["template"]["spec"]["nodeName"]  | <%= cb.nodes[0].name %> |
+      | ["items"][0]["spec"]["replicas"]                      | 1 |
     Then the step should succeed
     Given a pod becomes ready with labels:
       | name=test-pods |
     Then the step should succeed
     And evaluation of `pod(2).ip_url` is stored in the :p5pod1ip clipboard
     Given I save multus pod on node "<%= cb.nodes[1].name %>" to the :multuspod clipboard
+    Given I save multus pod on node "<%= cb.nodes[0].name %>" to the :multuspod0 clipboard
```

Comment:
list_for_pods is not created on specific node cb.nodes[0] now, but at this line, you still expect a multus pod on cb.nodes[0]?

Reply:
yes, here get the multus pod on different node to make sure different hostnetwork pods can access the list_for_pod.